safir/src/safir/testing/kubernetes.py

```diff
@@ -1840,6 +1845,176 @@ async def list_node(
         ]
         return V1NodeList(items=nodes)
 
+    # PERSISTENTVOLUME API
+
+    async def create_persistent_volume(
+        self,
+        body: V1PersistentVolume,
+        *,
+        _request_timeout: float | None = None,
+    ) -> None:
+        """Create a persistent volume.
+
+        Parameters
+        ----------
+        body
+            Persistent volume to create.
+        _request_timeout
+            Ignored, accepted for compatibility with the Kubernetes API.
+
+        Raises
+        ------
+        kubernetes_asyncio.client.ApiException
+            Raised with 409 status if the persistent volume already
+            exists.
+        """
+        self._maybe_error("create_persistent_volume", body)
+        self._update_metadata(body, "v1", "PersistentVolume", "__cluster__")
```

Comment:
I think this is still erroneously setting a `metadata.namespace` value of `__cluster__`.